lbz/lambdas/broker.py

```diff
@@ -26,11 +27,8 @@ def handle(self) -> LambdaResponse:
         if not (handler := self.mapper.get(op)):
             logger.error('No handler declared for requested operation: "%s"', op)
             return lambda_error_response(LambdaResult.CONTRACT_ERROR, f'"{op}" not implemented.')
-
         try:
-            if (data := self.raw_event.get("data")) is not None:
-                return handler(data)
-            return handler()
+            return handler(self.raw_event.get("data", {}))
```

Comment:
This won't protect us against everything but is a little bit safer if we always expect to get a dict: 
```suggestion
            return handler(self.raw_event.get("data") or {})
```